src/openforms/registrations/contrib/stuf_zds/tests/test_backend.py

```diff
@@ -1211,3 +1234,320 @@ def test_reference_can_be_extracted(self, m):
         reference = extract_submission_reference(submission)
 
         self.assertEqual("abcd1234", reference)
+
+
+@tag("gh-1183")
+@freeze_time("2020-12-22")
+@temp_private_root()
+class PartialRegistrationFailureTests(VariablesTestMixin, StUFZDSTestBase):
```

Comment:
Maybe this test should go in a new file, since it is already so big?

Reply:
Good shout!